app/lib/RouteResolver.js

```diff
@@ -33,13 +33,30 @@ function isRouteForRequest(route, req) {
 
   if (route.pathname !== '*' && !route.pathRegExp.test(pathname)) return false;
 
-  // TODO: Later add features to match other things, like query parameters, etc.
+  const matchesParams = _.every(route.query, (value, key) => {
+    return _.isEqual(_.get(req.query, key), value);
```

Comment:
I am fine with destructuring `lodash` methods. e.g. `const { get } = require('lodash')`.

Reply:
I can refactor. This PR or another?